Sofa/framework/Helper/src/sofa/helper/system/PluginManager.cpp

```diff
@@ -151,19 +159,21 @@ std::string PluginManager::getDefaultSuffix()
 #endif
 }
 
-bool PluginManager::loadPluginByPath(const std::string& pluginPath, std::ostream* errlog)
+auto PluginManager::loadPluginByPath(const std::string& pluginPath, std::ostream* errlog) -> PluginLoadStatus
 {
     if (pluginIsLoaded(pluginPath))
     {
-        return true;
+        const std::string msg = "Plugin '" + pluginPath + "' is already loaded";
+        if (errlog) (*errlog) << msg << std::endl;
```

Comment:
not sure you want to print an error for already loaded plugins. 
There will be a lot of print at each new start no?

Reply:
Actually, this line does not really print the message. It is up to the caller to print it or not. In this case, it is print in a message box, in case you want to load a plugin that is already loaded, from the plugin manager GUI. So, no flood here.

Reply:
yes but if you activate he logger to look for an error, you might have the real error in middle of plenty "already loaded". for me it is not an error.
But anyway, it was just a small remark. Nothing blocking.

Reply:
Activate what logger? I don't see how you can have plenty of "already loaded" messages. As I told you, this line does not print anything. It appends messages in a `std::ostream`.